{
  "comments": [
    {
      "robotId": "zuul",
      "robotRunId": "821b845ab601411e8b26e6f6ba8a99a2",
      "url": "https://telecominfraproject.zuul.vexxhost.dev/buildset/821b845ab601411e8b26e6f6ba8a99a2",
      "fixSuggestions": [],
      "key": {
        "uuid": "370a6a60_edd4e88a",
        "filename": "gnpy/core/elements.py",
        "patchSetId": 15
      },
      "lineNbr": 36,
      "author": {
        "id": 1017521
      },
      "writtenOn": "2021-09-16T08:01:12Z",
      "side": 1,
      "message": "linters-diff-ci: F401 \u0027gnpy.core.exceptions.ConfigurationError\u0027 imported but unused",
      "tag": "autogenerated:zuul:check",
      "revId": "1a483edbc2783900a370da271a5a89feafaeeb0a",
      "serverId": "d5d70762-12d0-45a1-890d-524b12d3f735"
    }
  ]
}